ci-operator/step-registry/ipi/conf/nutanix/external-lb/ipi-conf-nutanix-external-lb-commands.sh

```diff
@@ -0,0 +1,245 @@
+  mode tcp
+  balance roundrobin
+  option tcp-check
+  default-server verify none inter 10s downinter 5s rise 2 fall 3 slowstart 60s maxconn 250 maxqueue 256 weight 100
+EOF
+
+  for ip in {4..191}; do
+    ipaddress="$subnet_prefix"".""$ip"
+    echo "   "server "${EP_NAMES[$i]}"-"${ip}" "${ipaddress}":"${EP_PORTS[$i]}" check check-ssl >>"${haproxy_cfg}"
+  done
+done
+
+# scp haproxy.cfg to bastion host /tmp/haproxy.cfg
+scp -o UserKnownHostsFile=/dev/null -o IdentityFile="${SSH_PRIV_KEY_PATH}" -o StrictHostKeyChecking=no "${haproxy_cfg}" "${BASTION_SSH_USER}"@"${BASTION_IP}":${bastion_haproxy_cfg}
+
+# Reload haproxy.cfg by restart haproxy.service in bastion host
+ssh -o UserKnownHostsFile=/dev/null -o IdentityFile="${SSH_PRIV_KEY_PATH}" -o StrictHostKeyChecking=no "${BASTION_SSH_USER}"@"${BASTION_IP}" "sudo mkdir -p /etc/haproxy; sudo cp ${bastion_haproxy_cfg} /etc/haproxy/haproxy.cfg; sudo systemctl restart haproxy.service"
+
+# Update VIPs with external lb ip
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/install-config-external-lb.yaml"
+
+cat >"${PATCH}" <<EOF
+platform:
+  nutanix:
+    apiVIP: ${BASTION_IP}
+    ingressVIP: ${BASTION_IP}
+    loadBalancer:
+      type: UserManaged
+    # apiVIPs:
+    # - ${BASTION_IP}
+    # ingressVIPs:
+    # - ${BASTION_IP}
+EOF
+
+yq-go m -x -i "${CONFIG}" "${PATCH}"
+echo "Updated VIPs with external-lb in '${CONFIG}'."
+
+# Update DNS
+export API_VIP=$BASTION_IP
+export INGRESS_VIP=$BASTION_IP
+
+echo "${BASE_DOMAIN:?BASE_DOMAIN env variable should be defined}" > "${SHARED_DIR}"/basedomain.txt
+cluster_name="${NAMESPACE}-${UNIQUE_HASH}"
+base_domain=$(<"${SHARED_DIR}"/basedomain.txt)
+cluster_domain="${cluster_name}.${base_domain}"
+
+export AWS_SHARED_CREDENTIALS_FILE=/var/run/vault/nutanix/.awscred
+export AWS_MAX_ATTEMPTS=50
+export AWS_RETRY_MODE=adaptive
+export HOME=/tmp
+if ! command -v aws &> /dev/null
+then
+    echo "$(date -u --rfc-3339=seconds) - Install AWS cli..."
+    export PATH="${HOME}/.local/bin:${PATH}"
+    if command -v pip3 &> /dev/null
+    then
+        pip3 install --user awscli
+    else
+        if [ "$(python -c 'import sys;print(sys.version_info.major)')" -eq 2 ]
+        then
+          easy_install --user 'pip<21'
+          pip install --user awscli
+        else
+          echo "$(date -u --rfc-3339=seconds) - No pip available exiting..."
+          exit 1
+        fi
+    fi
+fi
+# shellcheck disable=SC1091
+source "${SHARED_DIR}/nutanix_context.sh"
+hosted_zone_id="$(aws route53 list-hosted-zones-by-name \
+            --dns-name "${base_domain}" \
+            --query "HostedZones[? Config.PrivateZone != \`true\` && Name == \`${base_domain}.\`].Id" \
+            --output text)"
+echo "${hosted_zone_id}" > "${SHARED_DIR}/hosted-zone.txt"
+# api-int record is needed just for Windows nodes
+# TODO: Remove the api-int entry in future
+echo "$(date -u --rfc-3339=seconds) - Creating DNS records ..."
+cat > "${SHARED_DIR}"/dns-create.json <<EOF
```

Comment:
I think we can reuse `ipi-conf-nutanix-dns`, just need to figure out way to pass down `BASTION_IP` to `ipi-conf-nutanix-dns`